VocabularyTrainer/Language Screens/LanguageScreenViewController.swift

```diff
@@ -70,6 +70,15 @@ class LanguageScreenViewController: UIViewController, UISearchBarDelegate, MFMai
         setGradientBackground(view: view)
         loadDataAndUpdate()
         sortVocabulary(element: .word, isAscending: true)
+        wordAddedObserver()
+    }
+    
+    private func wordAddedObserver() {
+        NotificationCenter.default.addObserver(self, selector: #selector(reloadData), name: NSNotification.Name(rawValue: "wordAdded"), object: nil)
```

Comment:
see https://github.com/misteu/VocabularyTraining/pull/23/commits/fef5d15de29b52ee9097259f9024eedda4475b1c#r1005699027

Please use the new extension then here as well :)

Reply:
Added :)